cita-jsonrpc/src/http_header.rs

```diff
@@ -192,9 +192,9 @@ mod test {
         vec![
             (Some("*"), HeaderValue::from_static("*")),
             (Some(" * "), HeaderValue::from_static("*")),
-            (None, HeaderValue::from_static("")),
-            (Some(""), HeaderValue::from_static("")),
-            (Some("   null "), HeaderValue::from_static("")),
```

Comment:
I do not understand this test case, could you help to explain for me?

Reply:
Test these:
https://github.com/cryptape/cita/blob/c8647954682bb68fd9707194692cb73c93ad2302/cita-jsonrpc/src/http_header.rs#L70-L76

Reply:
I see.